rest_framework/schemas/generators.py

```diff
@@ -135,6 +136,11 @@ def endpoint_ordering(endpoint):
     return (path, method_priority)
 
 
+_PATH_PARAMETER_COMPONENT_RE = re.compile(
+    r'<(?:(?P<converter>[^>:]+):)?(?P<parameter>\w+)>'
```

Comment:
This is the full regex copied from Django 2. I didn't dare to import it, since it had an underscore in front of it.